### ISSUE TITLE
Numbered sections

### ISSUE THREAD

**Issue by glupyan** [CrumpLab/NSFrmarkdown#1]
Hi, excellent work on this template. Thank you! How do I go about having numbered sections in the proj description. Looks like the LaTeX style overwrites the number_sections YAML parameter and I am not familiar enough with LaTeX to know how to fix it.

**Comment by CrumpLab** [comment]
Just looking into this now. Thanks to your question I also realized the page numbering at the bottom for each section of the grant (e.g., summary, project description, etc.) was incorrect because it didn't include lettering along with page number. I just updated the repo to fix that part. Will see about numbering sections within the project description.

EDIT: looks like the requirement to add letters for section along with pagination was a style requirement from an older proposal guide. Can be turned off by deleting the \newsection{} commands in the .rmd docs.

**Comment by CrumpLab** [comment]
Section numbering within a document should work now. I updated the repo to make the number_sections YAML work when set to true. 

The style of section headers is controlled by the nsf2.cls file in the style folder using the titlesec package. I found this [stackexchange post](https://tex.stackexchange.com/questions/25989/spacing-in-headline-between-number-and-text) helpful for changing the latex code for altering the style of headers with titlesec.
